src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1906,6 +1930,7 @@ private ArrayTypeSymbol VisitArrayInitializer(BoundArrayCreation node)
                                             useMethodSignatureParameterTypes: false,
                                             methodSignatureOpt: null,
                                             node,
+                                            binder: null,
```

Comment:
>binder: null, [](start = 44, length = 13)

Consider naming the parameter `binderOpt`

Reply:
I considered that but thought it might be misleading because `binder` is only `null` within `BestTypeForLambdaReturns`.

---
In reply to: [282703161](https://github.com/dotnet/roslyn/pull/35601#discussion_r282703161) [](ancestors = 282703161)